free-1.md

```diff
@@ -57,12 +70,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This seems like something in which language design would actually be the core of the work. I imagine extracting and manipulating the frequencies from a sound file is something that would probably be borrowed from another language or package. This would leave the programmer (me) with more time to actually think about and implement language design that makes sense.
```

Comment:
Something that might be worth thinking about is what kinds of sound files you're going to want to support. Although I haven't attempted to do any sound editing lately, I know dealing with different codecs can sometimes be a bigger pain than one might think.